ui.xml

```diff
@@ -222,7 +222,13 @@
     <Text id="panelReadyYes" visibility="Invisible" fontsize="36" color="#00FF00">Ready</Text>
     <Text id="panelReadyNo" visibility="Invisible" fontsize="36" color="#FF0000">Not Ready</Text>
 </Panel>
-<Panel id="panelUI" visibility="Invisible" width="160" height="140" rectAlignment="UpperRight" offsetXY="-240 0">
+<Panel id="panelPowerDraw" visibility="Invisible" allowDragging="true" width="120" height="60" rectAlignment="MiddleLeft" offsetXY="0 -40">
```

Comment:
do you have a screenshot of how this looks? because offhand it looks like those numbers will conflict with the player names in the top right corner?